main.coffee

```diff
@@ -179,6 +179,23 @@ americano._loadPlugins = (options, app, callback) ->
         callback()
 
 
+# Listen for http (or https) connections
+americano._listen = (app, options, callback) ->
+    if options.ssl
+        https = require 'https'
+        crtPath = process.env.SSL_CRT_PATH or '/etc/cozy/server.crt'
```

Comment:
Americano is supposed to not be tightly linked to cozy. May be allows CRT_PATH & KEY_PATH to be passed as options and move the `/etc/cozy` defaults to the cozy-proxy.